controller/errdefs/errdefs.proto

```diff
@@ -6,4 +6,5 @@ option go_package = "github.com/docker/buildx/controller/errdefs";
 
 message Build {
   string Ref = 1;
+  string BuildRef = 2;
```

Comment:
what's the difference?

Reply:
Can we rename this "kind" or "type" then?

Reply:
Looking a bit more at it, for local builds with controller it always set `local` but for remote it sets a random id that lives across the controller session: https://github.com/docker/buildx/blob/746eadd16e461ac1a961bb22e7fea8f1d05da88f/controller/remote/client.go#L120

So `ref` is still good but confusing as this is not a build ref. It's even more confusing as we already have `BuildOptions.Ref` in https://github.com/docker/buildx/blob/746eadd16e461ac1a961bb22e7fea8f1d05da88f/controller/pb/controller.proto#L80

I renamed it to `sessionID`.